pkg/kubelet/cm/types.go

```diff
@@ -35,6 +35,8 @@ type ResourceConfig struct {
 	HugePageLimit map[int64]int64
 	// Maximum number of pids
 	PodPidsLimit *int64
+	// OOM Kill Configuration
```

Comment:
Lets clarify the comment as something like "Configuration of out-of-memory process kill setting for pod"

Reply:
fixed